src/aura/BGE_EntryForm/BGE_EntryFormHelper.js

```diff
@@ -90,20 +90,34 @@
     },
 
     /**
-     * @description: queries open donations for upcoming donations
+     * @description: Gets open donations without any Payments and unpaid Payments
+     * for use in the Donation Selector component.
      * @return: void
      */
-    queryOpenDonations: function(component, donorId) {
+    getOpportunitiesWithOppPayments: function(component, recordId) {
         const donorType = component.get('v.donorType');
 
-        let action = component.get('c.getOpenDonations');
```

Comment:
With this change, do we need to keep the getOpenDonations() method in the apex controller around? Should we clean that up or do we still need that?

Reply:
And follow-up - it looks like there's several apex tests pointing to getOpenDonations, but we probably need similar coverage of the new method (and if we are responsibly removing that method, clearing out the dependent tests as well)

Reply:
Nope, the new `getOpportunitiesWithOppPayments` method completely replaces it.  I just pushed a round of commits that adds new test methods to cover the new method, and removes the old `queryOpenDonations` method and related test methods.